code/game/machinery/vending.dm

```diff
@@ -1518,6 +1518,7 @@ var/global/num_vending_terminals = 1
 		/obj/item/weapon/storage/pill_bottle/lollipops = 5,
 		/obj/item/weapon/reagent_containers/food/snacks/grown/potato = 100,
```

Comment:
what the fuck since when has getmore sold potatoes

Reply:
It does on St. Patrick's day, it is listed as an exclusive for St. Patricks day 29 lines below the one you highlighted.